semantic-conventions/src/tests/data/markdown/enum_extend_list/expected.md

```diff
@@ -0,0 +1,24 @@
+<!-- semconv db -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `db.system` | string |  | `"one"` | Recommended |
+
+`db.system` MUST be one of the following:
+
+| Value  | Description |
+|---|---|
+| `one` | one |
+<!-- endsemconv -->
+
+<!-- semconv second -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `first.enum_attr` | string |  | `"one"` | Recommended |
+
+`first.enum_attr` MUST be one of the following:
+
+| Value  | Description |
+|---|---|
+| `one` | one |
```

Comment:
Sorry maybe it's silly but where does `one` and `two` are coming from here? 🤔 I don't really get the `enum_extend_list` tests here. Seem they are also not used in any test?

Reply:
another great catch, removed them